compliance/cis_k8s/rules/cis_1_1_20/rule.rego

```diff
@@ -0,0 +1,18 @@
+package compliance.cis_k8s.rules.cis_1_1_20
+
+import data.compliance.lib.common
+import data.compliance.lib.data_adapter
+
+# Ensure that the Kubernetes PKI certificate file permissions are set to 644 or more restrictive (Manual)
+finding = result {
+	common.file_match_pattern("/etc/kubernetes/pki/[a-zA-Z]+.crt", data_adapter.file_path)
```

Comment:
Can't a CRT file name contain numbers? Also, I'm not a fan of regex here, can we avoid it? Why not validate the path and the filename separately?

Reply:
Sure, we can do it in two steps..